README.md

```diff
@@ -1,7 +1,62 @@
-# TeamFourHYF
-Recreation of HYF website
+# Project Definition
 
+## About
+
+With related to our course **Incremental Development** module and in practicing of **Planning & Collaboration on software projects**,we here by tried to **Recreate our school's** [website](https://hackyourfuture.be/).
+
+## Technologies
+
+The following tools were used in this project:
+
+- VScode
+- GitHub
+- Git
+- Html
+- CSS
+- Slack
+
+## General Steps
+
+  1. [Create and Setup GitHub Repository](https://github.com/lauraramirez220612/TeamFourHYF)
+  2. [Write Project Definition](README.md)
+  3. [Write Backlog](planning/backlog.md)
+  4. [Make Wire-frame](planning/wireframe.md)
+  5. [Development Strategy](planning/development-strategy.md)
+  6. [Development](https://github.com/lauraramirez220612/TeamFourHYF/blob/main/index.html)
```

Comment:
maybe we could put this instead? what do you think?
https://lauraramirez220612.github.io/TeamFourHYF/

Reply:
I agree that it would be better with the GitHub Pages link.